tests/modules/programs/goimapnotify/default.nix

```diff
@@ -0,0 +1 @@
+{ goimapnotify-launchd = ./launchd.nix; }
```

Comment:
I've only tested this on darwin — I don't know whether the `launchd` tests run on linux (I can test this, though).

Reply:
Ah, tests are passing in CI.